src/query/sql/src/executor/format.rs

```diff
@@ -94,24 +94,39 @@ fn table_scan_to_format_tree(
                 .map_or("NONE".to_string(), |limit| limit.to_string())
         });
 
+    let mut children = vec![
+        FormatTreeNode::new(format!("table: {table_name}")),
+        FormatTreeNode::new(format!("read rows: {}", plan.source.statistics.read_rows)),
+        FormatTreeNode::new(format!("read bytes: {}", plan.source.statistics.read_bytes)),
+        FormatTreeNode::new(format!(
+            "partitions total: {}",
+            plan.source.statistics.partitions_total
+        )),
+        FormatTreeNode::new(format!(
+            "partitions scanned: {}",
+            plan.source.statistics.partitions_scanned
+        )),
+        FormatTreeNode::new(format!(
+            "push downs: [filters: [{filters}], limit: {limit}]"
+        )),
+    ];
+
+    let mut output_columns: Vec<usize> = Vec::new();
+    if let Some(scan_fields) = &plan.source.scan_fields {
+        output_columns = scan_fields.keys().cloned().collect();
```

Comment:
I don't know when and why `output_columns` is removed from explain.

I think it's useful to help us check if prune column works well.

Reply:
Good to do.
BTW, `output_columns` is a slice?

Reply:
Yes